src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -15,8 +15,6 @@
  *    limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
```

Comment:
Seems to me that if this file declares interfaces inheriting from NSObject, it needs to import Foundation.

Reply:
It seems OK to me to rely on MTRDefines importing Foundation?

Reply:
OK.